package.json

```diff
@@ -9,8 +9,9 @@
     ".": {
       "types": "./lib/index.d.ts",
       "require": "./lib/index.js",
-      "default": "./lib/index.mjs"
-    }
+      "default": "./lib/esm/index.js"
```

Comment:
We don't build into this file. I think you meant `./lib/index.mjs`, which we do emit. 

I will provide adjustments to this right now.